scripts/synapse_port_db

```diff
@@ -128,10 +128,20 @@ APPEND_ONLY_TABLES = [
 
 
 IGNORED_TABLES = {
+    # We don't port these tables, as they're a faff and we can regenerate
+    # them anyway.
     "user_directory",
     "user_directory_search",
-    "users_who_share_rooms",
-    "users_in_pubic_room",
+    "user_directory_search_content",
+    "user_directory_search_docsize",
+    "user_directory_search_segdir",
+    "user_directory_search_segments",
+    "user_directory_search_stat",
+    "user_directory_search_pos",
```

Comment:
I'm not sure if all of these are regenerated or not...I might be totally off base here.

Reply:
I think a bunch of these are actually sqlite specific stuff anyway (to support full text search)

Reply:
That means we should include them, correct?

Reply:
Indeed